modules/ldap-syncing-running-all-ldap.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing-groups.adoc
+
+[id="ldap-syncing-running-all-ldap_{context}"]
+= Syncing the LDAP server with {product-title}
+
+You can sync all groups from the LDAP server with {product-title}.
+
+.Prerequisite
```

Comment:
Should be Prerequisites plural, even if it's only one. A few other places as well.